src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/ShellRenderer.cs

```diff
@@ -172,6 +180,7 @@ void OnLayout()
             if (_currentItem != null)
             {
                 _currentItem.Geometry = _mainLayout.Geometry;
+                _currentItem.StackAbove(_drawer);
```

Comment:
_currentItem will above than _drawer, is it intended?

Reply:
_drawer could be null

Reply:
Fixed.